ko/csql.rst

```diff
@@ -815,3 +815,10 @@ CSQL 인터프리터에서 작업 중인 데이터베이스 이름 및 호스트
 
     csql> ;editor_cmd emacs
     csql> ;edit
+
+**싱글 라인 모드 설정(;SIngleline)**
+
+싱글 라인 모드를 **ON** 또는 **OFF** 로 설정하는 명령어이다 (기본값 **OFF**). 싱글 라인 모드 ON에서는 세미콜론(;)과 ENTER를 입력하면 SQL문이 바로 실행된다. 싱글 라인 모드 OFF에서는 여러 개의 SQL 문을 저장해 두었다가 ;xr 혹은 ;r 세션 명령어로 한꺼번에 수행한다. 만약, ON 또는 OFF 를 지정하지 않으면 현재 설정된 값을 보여준다. ::
```

Comment:
> 만약, ON 또는 OFF 를 지정하지 않으면 현재 설정된 값을 보여준다.

@kisoo-han 
This sentence is added like ;autocommit command

Reply:
It follows the convention of most csql session commands. They mostly do not explain the absence of options, but show examples of use, it is intuitive. And because OFF is default, you'd better use ON as an example,

csql> ;singleline on
SINGLE IS ON
csql> ;signleline
SINGLE IS ON

Reply:
@kisoo-han 
I knew that ON is default. ([http://jira.cubrid.org/browse/CBRD-24599](http://jira.cubrid.org/browse/CBRD-24599)

I was also worried about the intuition. But I guess there might be requests about editing the contents.
So, I add it the contents about the no option case.

I will add no option case manual example.

Reply:
TS team wanted us to set the default of this command to ON.
please change "(기본값 OFF)" to "(기본값 ON)".

Reply:
Ok, I changed as 'ON'. Thank you.